packages/contracts-bedrock/src/dispute/lib/Types.sol

```diff
@@ -70,6 +70,9 @@ library GameTypes {
     /// @notice A dispute game type that uses OP Succinct
     GameType internal constant OP_SUCCINCT = GameType.wrap(6);
 
+    /// @notice A dispute game type that uses the cannon vm (version 2).
+    GameType internal constant CANNON_2 = GameType.wrap(7);
+
```

Comment:
I don't think we want to create a new game type for this.  Should just use the existing CANNON type.